app/src/main/AndroidManifest.xml

```diff
@@ -1,131 +1,193 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="org.gdg.frisbee.android" android:versionCode="27" android:versionName="1.0.10" android:installLocation="auto">
+  package="org.gdg.frisbee.android"
+  android:installLocation="auto">
 
-    <uses-feature android:name="android.hardware.nfc" android:required="false" />
+  <uses-feature
+    android:name="android.hardware.nfc"
+    android:required="false" />
 
-    <supports-screens android:normalScreens="true" android:smallScreens="true" android:largeScreens="true" android:xlargeScreens="true"/>
-    <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="com.google.android.c2dm.permission.RECEIVE" />
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.USE_CREDENTIALS"/>
-    <uses-permission android:name="android.permission.GET_ACCOUNTS" />
-    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
-    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
-    <uses-permission android:name="android.permission.WAKE_LOCK" />
-    <uses-permission android:name="android.permission.NFC" />
-    <uses-permission android:name="android.permission.GET_TASKS" />
+  <supports-screens
+    android:normalScreens="true"
```

Comment:
I am told that using this is wrong. Because it disables out some devices and possible devices in the future. Like xxlarge screens for example.